File: launchdarkly/ruby/src/GetInsightsScoresExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["ApiKey"] = "YOUR_API_KEY"
end

begin
    response = LaunchDarklyClient::InsightsScoresBetaApi.new.get_insights_scores(
        nil, # project_key
        nil, # environment_key
    )

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling InsightsScoresBeta#get_insights_scores: #{e}"
end

File: launchdarkly/python/src/GetInsightsScoresExample.py
import json
from datetime import date, datetime
from pprint import pprint

from launchdarkly_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"ApiKey": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    try:
        response = api.InsightsScoresBetaApi(api_client).get_insights_scores(
            project_key=None,
            environment_key=None,
            application_key=None,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling InsightsScoresBetaApi#get_insights_scores: %s\n" % e)
